AnkiDroid/src/main/java/com/ichi2/libanki/Collection.java

```diff
@@ -367,7 +366,7 @@ public String loadColumn(String columnName) {
                 }
                 String res = cursor.getString(0);
                 if (res.length() == 0) {
-                      break;
+                    break;
```

Comment:
```suggestion
                      break;
```

it's not fully reverted yet. This restores the line to the previous state so it will not appear in the diff I think